src/olympia/files/admin.py

```diff
@@ -35,14 +35,14 @@ class FileAdmin(admin.ModelAdmin):
         }),
         ('Details', {
             'fields': (
-                'jetpack_version', 'cert_serial_num', 'original_status'),
+                'cert_serial_num', 'original_status'),
         }),
         ('Flags', {
             'fields': (
                 'is_restart_required', 'strict_compatibility',
-                'requires_chrome', 'binary', 'binary_components',
-                'is_signed', 'is_multi_package', 'is_experiment',
```

Comment:
A follow-up issue at least, but is `is_multi_package` still used?  multiple packaged addons were a legacy only thing, iirc.

Reply:
You're right it's not, I'll file an issue and take care of it.

Reply:
https://github.com/mozilla/addons-server/issues/10978